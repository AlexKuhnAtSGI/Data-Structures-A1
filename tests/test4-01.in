i 
seek
out 
odds
yet 
these 
all 
are 
evens 
odd
odd
odd
odd
odd
odd
odd
odd
odd