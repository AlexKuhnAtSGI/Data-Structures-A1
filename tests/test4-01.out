i 
seek
out 
odds
yet 
these 
all 
are 
evens 
